client/src/containers/Referrals/Referrals.js

```diff
@@ -2,34 +2,71 @@ import React, { Component } from "react";
 import * as api from "../../helpers/api";
 import "./Referrals.css";
 
-class Referrals extends Component {
+class Referrals extends React.Component {
   state = {
-    referrals: []
+    firstName: "",
+    lastName: "",
+    email: "",
+    phoneNumber: "",
+    address: "",
+    howLongBeenHomeless: ""
+  };
+  onSubmigit  = e => {
+    e.preventDefault();
+    console.log(this.state);
   };
-  componentDidMount() {
-    api.getReferrals().then(data => {
-      this.setState({
-        referrals: data
-      });
-    });
-  }
   render() {
     return (
-      <div>
-        <h1>List of referrals</h1>
-        <div class="referrals-container">
-          {this.state.referrals.map(referral => {
-            return (
-              <div className="referral-info">
-                <h3>{referral.name}</h3>
-                <p>{referral.city}</p>
-              </div>
-            );
-          })}
-        </div>
-      </div>
+      <form>
+        <input
+          placeholder="First name"
+          value={this.state.firstName}
+          onChange={e => this.setState({ firstName: e.target.value })}
+        />
+        <br />
+        <input
+          placeholder="Last name"
+          value={this.state.lastName}
+          onChange={e => this.setState({ lastName: e.target.value })}
+        />
+        <br />
+        <input
+          placeholder="Age"
+          onChange={e => this.setState({ Age: e.target.value })}
+        />
+        <br />
+        <input
+          placeholder="Email"
+          value={this.state.email}
+          onChange={e => this.setState({ email: e.target.value })}
+        />
+        <br />
+        <input
+          placeholder="Phone number"
+          value={this.state.phoneNumber}
+          onChange={e => this.setState({ phoneNumber: e.target.value })}
+        />
+        <br />
+        <input
+          placeholder="address"
+          value={this.state.password}
+          onChange={e => this.setState({ password: e.target.value })}
```

Comment:
phoneNumber and password are not part of the initial state, but you're setting them in here. Could we either remove them in all places or keep them please?